### ISSUE TITLE
Cannot copy key after adding second Azure account

### ISSUE THREAD

**Issue by peef1981** [microsoft/AzureStorageExplorer#7223]
### Preflight Checklist

- [X] I have installed the [latest version of Storage Explorer](https://github.com/Microsoft/AzureStorageExplorer/releases/latest).
- [X] I have checked existing resources, including the [troubleshooting guide](https://docs.microsoft.com/azure/storage/common/storage-explorer-troubleshooting) and the [release notes](https://github.com/microsoft/AzureStorageExplorer/releases).
- [X] I have searched for [similar issues](https://github.com/Microsoft/AzureStorageExplorer/issues).

### Storage Explorer Version

1.21.1

### Architecture

x64

### Storage Explorer Build Number

20210921.5

### Platform

macOS

### OS Version

macOS Ventura 13.5.1

### Bug Description

Unable to right-click and copy primary key for a storage account after adding a second Azure subscription

### Steps to Reproduce

1. start with a single Azure account
2. pick a storage account
3. right-click and copy Primary Key
4. add a second Azure account
5. pick a storage account
6. able to right-click and copy Primary Key from this second account
7. receive the error in context section below when using first account

### Actual Experience

See Steps. I've never had a problem copying the key before but only ever had a single Azure account. I recently added a second free trial to perform some experiments, but when I needed to go back and copy key from first account it failed. A colleague who happened to only have a single Azure account was able to copy it for me.

### Expected Experience

I expect to be able to copy key from any storage account I have access to

### Additional Context

Error Message

```
The requested action could not be completed.

Details:
Error processing argument at index 0, conversion failure from undefined
```

**Comment by JasonYeMSFT** [comment]
@peef1981 According to our support policy, 1.21.1 is no longer supported [Storage Explorer support policy](https://learn.microsoft.com/en-us/azure/storage/common/storage-explorer-support-policy-lifecycle#releases). I recommend you to upgrade if it is possible. If it's not, I think you can check if your Azure user still has the permissions to list storage account keys for that specific storage account.

**Comment by peef1981** [comment]
I got "no upgrade available" when I did "Check for updates" within 1.21.1 but found 1.31.1 when I went online. I no longer get the error message, but I get different behavior when I try and copy key from the two subscriptions. This leads me to believe something has actually changed on at least one of these accounts and I will check w/ our Platform Tech team. Thank you.
